int32 indexChannel
string description
float32 relevance
int32 layer